tests/unit/metrics-recorder.unit.ts

```diff
@@ -0,0 +1,33 @@
+import { expect } from '@loopback/testlab'
+
+describe('Metrics recorder (unit)', () => {
```

Comment:
Could you clarify the reason for these tests? They do not seem to actually test the metrics recorder code.

Reply:
> Could you clarify the reason for these tests? They do not seem to actually test the metrics recorder code.

That means the naming needs to change. The purpose is this test was only to test the small piece reducing the methods.

Reply:
The naming solved in e924cd7db02ab031196157788add8228940920bb